src/components/shared/MembersList/MembersList.tsx

```diff
@@ -1,48 +1,45 @@
 import React, { ReactNode } from 'react';
 
 import ListGroup, { ListGroupAppearance } from '~shared/ListGroup';
-import { AnyUser, Colony } from '~data/index';
+import { Watcher, Contributor } from '~types';
 
 import MembersListItem from './MembersListItem';
+import { notNull } from '~utils/arrays';
 
-interface Props<U> {
-  colony: Colony;
-  extraItemContent?: (user: U) => ReactNode;
-  onRowClick?: (user: U) => void;
+interface Props {
+  users: (Watcher | Contributor)[];
```

Comment:
It looks like we use `Watcher | Contributor` quite a lot, maybe it's worth creating a dedicated type for it, e.g. `type Member = Watcher | Contributor`